ivy/functional/ivy/general.py

```diff
@@ -2262,10 +2262,28 @@ def set_queue_timeout(timeout):
 
 
 def queue_timeout():
```

Comment:
This function need not be formatted in this PR, can you please remove the changes as it's causing `docstrings` to fail

Reply:
Hi @rush2406, thanks for your message. I will add these methods soon.

Reply:
Hi @rush2406 and @ankitdsi2010 , 

thank you very much for your comments. I made the changes. However, the docstring tests fail due to some issues in /functional/ivy/activations.py (relu(), line 55). To test, I run in Docker schell: 

>>> x = ivy.array([1.5, 0.7, -2.4])
>>> y = ivy.zeros(3)

It results in:

  File "/ivy/ivy/functional/ivy/creation.py", line 185, in zeros
    return current_backend().zeros(shape, dtype=dtype, device=device, out=out)
TypeError: zeros() got an unexpected keyword argument 'out'.

It seems to be unrelated to the inplace_decrement(), and it is OK to do changes in the  relu() in this PR. Please advice. 

Thank you very much. 

Best regards, Elena.

Reply:
@edaehn It's fine, you need not worry about the failures from other functions

Reply:
Great! Thank you very much @rush2406